lib/api/request/kuzzleRequest.ts

```diff
@@ -1007,6 +1001,21 @@ export class KuzzleRequest {
 
     return value;
   }
+
+  /**
+   * Generic getter value: date value.
+   *
+   * @param name parameter name
+   * @private
+   */
+  private _getDate (name: string): Date {
+    const args = this.input.args;
+    if (args[name] === undefined) {
+      throw assertionError.get('missing_argument', name);
+    }
+    return new Date(args[name]);
```

Comment:
What if the format is incorrect?

Reply:
this function is dropped since I don't cast to Date now